docs/source/coding_conventions.rst

```diff
@@ -0,0 +1,119 @@
+Coding Conventions
+==================
+
+In order to increase readability across Tensor Comprehensions written by
+multiple authors and to reduce the amount of surprising behavior, the
+following conventions should be adopted when writing TC. Generally in TC one
+should increment nesting by 4 whitespaces at each level and align tensor names
+and indices where appropriate to make memory access patterns emerge. Since
```

Comment:
does this mean that inline whitespace is encouraged?  mention this specifically?

Is this example what you want?  Do we encourage inline whitespace on both LHS and RHS?
```
A(i,j+1) += B(i, j   , k)
C(i,j)   += B(i, j-1, k)
```

Reply:
unspecified, use your best judgement, I'd certainly not limit whitespacing to only RHS.